### PR TITLE
S3t2 create user page

### PR DESCRIPTION
**Pull request by nico-dev-it**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+  font-family: "Trebuchet MS", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #ECEFF8;
+}
+body nav {
```
Comment (JVLoe) [inline index.css:10]: space between styling

File: index.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  background-color: #4F9CC8;
+  padding: 10px;
+  position: fixed;
+  width: 100vw;
+}
+body h1 {
```
Comment (JVLoe) [inline index.css:16]: space between styling

File: index.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  padding-left: 10px;
+  color: white;
+}
+body .user-header {
```
Comment (JVLoe) [inline index.css:20]: space between styling

File: index.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  padding: 15px;
+  position: relative;
+  padding-top: 70px;
+}
+body .userCard {
```
Comment (JVLoe) [inline index.css:25]: space between styling

File: index.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  background-color: #9DBED4;
+  width: 400px;
+  border: solid 2px #4F9CC8;
+  padding: 10px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+body .avatarImg {
```
Comment (JVLoe) [inline index.css:35]: space between styling

File: index.css [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  max-height: 200px;
+  max-width: 200px;
+  clip-path: circle(40% at 50% 50%);
+  align-self: center;
+}
+body .cardUsernameText {
```
Comment (JVLoe) [inline index.css:41]: space between styling

File: src/User.php [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        return $this->avatar;
+    }
+
+    public function getBio(): string
+    {
+        return $this->bio;
+    }
+
+    public function getHashedPass(): string
```
Comment (JVLoe) [inline src/User.php]: public function getHashedPass(): string keeps slipping into the code 😭 must remove pls

File: user_profile.css [hunk 8]
```diff
@@ -0,0 +1,41 @@
+@import './index.css';
+.userProfileContainer {
+  padding: 50px 10px 0 10px;
+}
+.userProfileContainer a {
```
Comment (JVLoe) [inline user_profile.css]: space between styling

File: user_profile.css [hunk 9]
```diff
@@ -0,0 +1,41 @@
+@import './index.css';
+.userProfileContainer {
+  padding: 50px 10px 0 10px;
```
Comment (JVLoe) [inline user_profile.css]: 0px or just 0?

File: user_profile.css [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+}
+.userProfileContainer a {
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 1.3rem;
```
Comment (JVLoe) [inline user_profile.css]: do we need to set a standard font size before using rem?

File: user_profile.css [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+  align-items: center;
+}
+.userProfileContainer .bioTextContainer {
+  border: 1px solid #D1D8E2;
+  border-radius: 15px;
+  background-color: #D1D8E2;
+  max-width: 600px;
+  overflow-wrap: break-word;
+  padding: 10px;
+  margin: 50px 10px 0 10px;
```
Comment (JVLoe) [inline user_profile.css]: 0 or 0px?

File: user_profile.php [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+                I have many interests including coding, philosophy, politics, Magic the Gathering, Dungeons and Dragons, spicy Reddit discussions, gardening, burritos, coding, philosophy, politics, Magic the Gathering, Dungeons and Dragons, spicy Reddit discussions, gardening, burritos, and many more. I care a lot about the environment and about making the world a better place for everyone.
+
+                When it comes to coding, I'm more of a backend guy, but I have been known to dedicate many hours to CSS styling for the greater benefit of the team. I look forward to coding more code in the near future, hopefully less CSS!
+            </p>
+        </div>
+    </div>
+    </p>
+</div>
+</div>
```
Comment (JVLoe) [inline user_profile.php]: one too many closing divs?

File: user_profile.php [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+        <div class="bioTextContainer">
+            <p>Hello, I am the one and only Mike Oram, student at iO Academy.
+
+                I have many interests including coding, philosophy, politics, Magic the Gathering, Dungeons and Dragons, spicy Reddit discussions, gardening, burritos, coding, philosophy, politics, Magic the Gathering, Dungeons and Dragons, spicy Reddit discussions, gardening, burritos, and many more. I care a lot about the environment and about making the world a better place for everyone.
+
+                When it comes to coding, I'm more of a backend guy, but I have been known to dedicate many hours to CSS styling for the greater benefit of the team. I look forward to coding more code in the near future, hopefully less CSS!
+            </p>
+        </div>
+    </div>
+    </p>
```
Comment (JVLoe) [inline user_profile.php]: what is this closing ?

File: user_profile.scss [hunk 14]
```diff
@@ -0,0 +1,45 @@
+@import './index.css';
+
+$text-color-black: #000000;
+$background-color-text: #D1D8E2;
+$breakpoint-desktop: 750px;
+
+.userProfileContainer {
+  padding: 50px 10px 0 10px;
```
Comment (JVLoe) [inline user_profile.scss]: 0 or 0px?

File: user_profile.scss [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+  a {
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 1.3rem;
+    color: $text-color-black;
+  }
+  h1 {
```
Comment (JVLoe) [inline user_profile.scss]: space between stylings

File: user_profile.scss [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+$text-color-black: #000000;
+$background-color-text: #D1D8E2;
+$breakpoint-desktop: 750px;
+
+.userProfileContainer {
+  padding: 50px 10px 0 10px;
+  a {
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 1.3rem;
```
Comment (JVLoe) [inline user_profile.scss]: is rem referring to a standard font size?

File: user_profile.scss [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+    align-items: center;
+  }
+  .bioTextContainer {
+    border: 1px solid #D1D8E2;
+    border-radius: 15px;
+    background-color: $background-color-text;
+    max-width: 600px;
+    overflow-wrap: break-word;
+    padding: 10px;
+    margin: 50px 10px 0 10px;
```
Comment (JVLoe) [inline user_profile.scss]: 0 or 0px?

**Review by JVLoe — CHANGES_REQUESTED** [review]
index.css 

- space between each styling 
- creators discretion: putting each  selector on a new line for readability?

User.php 
- Sorry, public function getHashedPass(): string keeps slipping into pushed code - must remove 

user_profile.css

using snake_case - change to camel case ---- user_profile.php too 
space between styling for readability

File: src/User.php (hunk 7 above)
Comment (nico-dev-it) [reply src/User.php]: 🤯 hopefully it is gone gone now...

**Pull request merged**